### PR TITLE
Diqing dev

### PR DESCRIPTION
**Pull request by changdiqing**
redux sample code:
 - ReduxActions
 - store

new component:
 - ShoppingPage

### CODE REVIEW

File: src/redux/cartReducers.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {CART_ADD_ITEM, CART_REMOVE_ITEM} from './reduxConstants';
+
+function cartReducer(state={cartItems: []}, action){
+
+    const payload = action.payload;
+
+    switch(action.type){
+        case CART_ADD_ITEM:
+            console.log("here it beginns");
```
Comment (JonlineRB) [inline src/redux/cartReducers.js:9]: can remove logs when done testing

File: src/components/ShoppingPage.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
+:root {
+    --red: #b00;
+    --blue: #00b;
+    --darkGreen: #036635;
+    --lightGreen: #01B636;
+    --grey: #E3E3E3;
+  }
```
Comment (MarcoTUM) [inline src/components/ShoppingPage.css:7]: Need to be in it's own css file.

File: src/views/ShopSelectionView.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    </Col>
+                </Row>
+            
+                
+          </main>
+        );
+    }
+}
+
+const sideBarStyle = {
```
Comment (JonlineRB) [inline src/views/ShopSelectionView.js:112]: In file css, move this to an external css file?

File: src/redux/cartReducers.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            console.log(state);
+            console.log(payload);
```
Comment (MarcoTUM) [inline src/redux/cartReducers.js:11]: Please remove console logs once you don't need them anymore.

File: src/views/ShopSelectionView.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+const yellowBold = {
+    fontWeight: 'bold',
+    color: 'yellow',
+    textAlign:"left"
+};
+
+const yellow = {
+    color: 'yellow',
+    textAlign:"right"
+}
```
Comment (MarcoTUM) [inline src/views/ShopSelectionView.js:134]: Move to css file.

**Pull request merged**